bitnami/contour/Chart.yaml

```diff
@@ -25,4 +25,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/containers/tree/main/bitnami/contour
   - https://projectcontour.io
-version: 10.1.5
+version: 10.1.6
```

Comment:
As you are adding new parameters, please update the version to `10.2.0`.

Reply:
Done.